src/clims/services/substance.py

```diff
@@ -73,13 +78,30 @@ def all_qs(self):
 
         return SubstanceVersion.objects.filter(latest=True).prefetch_related('properties')
 
+    def all_submission_files(self, organization):
+        # TODO: Currently returns OrganizationFile. Need to filter it down to only substance files
+        #       So add a "type" to the file.
+        return OrganizationFile.objects.filter(organization=organization)
+
+    def get_submission_file(self, file_id):
+        try:
+            return OrganizationFile.objects.get(id=file_id)
+        except OrganizationFile.DoesNotExist:
+            raise NotFound("Can't find file with id '{}'".format(file_id))
+
     @transaction.atomic
-    def load_file(self, organization, full_path, fileobj):
+    def load_file(self, organization, full_path, fileobj, add_timestamp=False):
+        # TODO: Decide what we want to happen here. Should the user be able to load files
```

Comment:
I agree. The file name shouldn't matter. The file content is what should be checked and validated to not add e.g. duplicate samples.